Examples/ObjectiveC/ManageOfflineRegionsExample.m

```diff
@@ -3,12 +3,281 @@
+    float progressPercentage = (float)completedResources / expectedResources;
+    
+    // At this point, the offline pack has finished downloading.
+    if (pack.state == MGLOfflinePackStateComplete ) {
+        
+        self.progressView = [[UIProgressView alloc] initWithProgressViewStyle:UIProgressViewStyleDefault];
+        CGSize frame = self.view.bounds.size;
+        self.progressView.frame = CGRectMake(frame.width / 4,
+                                             frame.height * 0.75f,
+                                             frame.width / 2, 10);
+        [self.view addSubview:self.progressView];
+    }
+
+    [self.progressView setProgress:progressPercentage animated:YES];
+
+    // If this pack has finished, print its size and resource count.
+    if (completedResources == expectedResources) {
+        NSString *byteCount = [NSByteCountFormatter
+                               stringFromByteCount:progress.countOfBytesCompleted countStyle:NSByteCountFormatterCountStyleMemory];
+        NSLog(@"Offline pack “%@” completed: %@, %llu resources",
+              userInfo[@"name"], byteCount, completedResources);
+    } else {
+        // Otherwise, print download/verification progress.
+        NSLog(@"Offline pack “%@” has %llu of %llu resources — %.2f%%.",
+              userInfo[@"name"], completedResources,
+              expectedResources, progressPercentage * 100);
+    }
+
+    // Reload the table to update the progress percentage for each offline pack.
+    [self.tableView reloadData];
+
+}
+
+#pragma mark - UITableView configuration
+
+// Create the table view which will display the downloaded regions.
+- (NSInteger)tableView:(nonnull UITableView *)tableView
+ numberOfRowsInSection:(NSInteger)section {
+    if (MGLOfflineStorage.sharedOfflineStorage.packs.count != 0) {
+        return MGLOfflineStorage.sharedOfflineStorage.packs.count;
+    } else {
+        return 0;
+    }
+
+}
+
+- (UIView *)tableView:(UITableView *)tableView
+viewForHeaderInSection:(NSInteger)section {
+    UILabel * label = [[UILabel alloc] init];
+    label.backgroundColor = UIColor.systemBlueColor;
+    label.textColor = UIColor.whiteColor;
+    label.font = [UIFont preferredFontForTextStyle: UIFontTextStyleHeadline];
+    label.textAlignment = NSTextAlignmentCenter;
+
+
+    if (MGLOfflineStorage.sharedOfflineStorage.packs != nil) {
+        label.text = @"Offline maps";
+    } else {
+        label.text = @"No offline maps";
+    }
+
+    return label;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView
+heightForHeaderInSection:(NSInteger)section {
+    return 50.0;
+}
+
+- (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView
+                 cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath {
+
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"cell" forIndexPath:indexPath];
+
+    [cell initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:@"cell"];
```

Comment:
The cell is being initialized twice here. Instead, you can do this:

```suggestion
    UITableViewCell *cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle
                                                 reuseIdentifier:@"cell"];
```

(Note - I also aligned the function call along its parameter colons)